db/migrate/20230201151352_add_qualified_teaching_status_to_jobseeker_profiles.rb

```diff
@@ -0,0 +1,5 @@
+class AddQualifiedTeachingStatusToJobseekerProfiles < ActiveRecord::Migration[7.0]
+  def change
+    add_column :jobseeker_profiles, :qualified_teaching_status, :string
```

Comment:
I wonder if this should be an enum instead?

To do so, in the JobseekerProfile model you'd have something like:

`enum qualified_teacher_status: { yes: 0, no: 1, on_track: 2 }`

Then, the migration will look like this:

`add_column :vacancies, :qualified_teacher_status, :integer`